combinations/hash.tsv

```diff
@@ -442,8 +442,8 @@ minimap2=2.26,samtools=1.17
 r-tidyverse=2.0.0,r-knitr=1.43,r-dt=0.28,r-formattable=0.2.1,r-purrr=1.0.1,r-vegan=2.6_4,r-rmarkdown=2.22,r-optparse=1.7.3,r-ggplot2=3.4.2,r-dplyr=1.1.2,r-data.table=1.14.8,r-base=4.2.3,r-dtplyr=1.3.1,r-patchwork=1.1.2	quay.io/bioconda/base-glibc-debian-bash:latest	1
 r-propr=4.2.6
 r-propr=5.0.4
-r-propr=5.1.0,bioconductor-limma=3.58.1
```

Comment:
you can leave the old line in the document

Reply:
that version is not very useful though, it has some bugs, etc.
and that version was not used by nf-core modules yet.

Reply:
If you remove a line, you are not removing a container. Just the trace in the file. Whether it is used by nf-core or not is not relevant to this repo. But it also does not matter, I will merge now.

What strikes me a bit is that nf-core seems to add containers here that are not used and are broken :( which is a problem! Someone needs to pay the storage.